middlewares/authorizationJwt.js

```diff
@@ -0,0 +1,40 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const service = require("../service/users.js");
+require("dotenv").config();
+const secret = process.env.SECRET;
+
+const extractJWT = passportJWT.ExtractJwt;
+const Strategy = passportJWT.Strategy;
+
+const params = {
+  secretOrKey: secret,
+  jwtFromRequest: extractJWT.fromAuthHeaderAsBearerToken(),
+};
+
+passport.use(
+  new Strategy(params, async (payload, done) => {
+    try {
+      const user = await service.getUserById(payload.id);
+      if (!user) return done(new Error("User not found"));
+      if (user) {
+        return done(null, user);
+      }
```

Comment:
to jest dziwne, bo traktujesz taki sam kod na 2 rozne sposoby. Dlaczego pierwszy nie ma `{}`, ale drugi juz ma? Potraktuj je tak samo.